### PR TITLE
Added deprecation info to snapshot sources property

### PR DESCRIPTION
**Pull request by core23**
Never used like the same property in the ``Page`` model.

### CODE REVIEW

**Comment by Soullivaneuh** [comment]
Please add `trigger_error` function inside deprecated methods.

**Comment by core23** [comment]
Done @Soullivaneuh

File: Model/Snapshot.php [hunk 1]
```diff
@@ -387,18 +390,22 @@ public function getParentId()
     }
 
     /**
-     * {@inheritdoc}
+     * @deprecated This method is deprecated since version 2.4 and will be removed in 3.0.
```
Comment (Soullivaneuh) [inline Model/Snapshot.php]: Remove "This method is deprecated". `@deprecated` already says that.

File: Model/Snapshot.php [hunk 2]
```diff
@@ -387,18 +390,22 @@ public function getParentId()
...
     public function setSources($sources)
     {
+        trigger_error('The '.__METHOD__.' method is deprecated since version 2.4 and will be removed in 3.0.', E_USER_DEPRECATED);
+
         $this->sources = $sources;
     }
 
     /**
-     * {@inheritdoc}
+     * @deprecated This method is deprecated since version 2.4 and will be removed in 3.0.
```
Comment (Soullivaneuh) [inline Model/Snapshot.php]: Remove "This method is deprecated". `@deprecated` already says that.

**Comment by core23** [comment]
Changed depreaction text

**Comment by OskarStark** [comment]
ping @Soullivaneuh

**Pull request merged**